previewer/main_renderer.js

```diff
@@ -71,6 +71,15 @@ $(document).ready(function(){
               )
         )
       )
+
+      // give a 'curent' class to clicked items the documentList
+      $('tr').on('click', function(){
+        $(this)
+        .addClass('current')
+        .siblings('.current')
```

Comment:
Split this out for easier reading - one line for adding the class and one to remove it from other siblings.